src/lib/JWT.js

```diff
@@ -0,0 +1,54 @@
+import jwt from "jsonwebtoken";
+import TokenExpiredError from "../errors/TokenExpiredError";
+import JsonWebTokenError from "../errors/JsonWebTokenError";
+
+export const generateToken = ({ firstName, lastName }) => {
+  // Let's reuse SECRET_COOKIE_PASSWORD instead of creating another one
+  const token = jwt.sign(
+    { firstName, lastName },
+    process.env.SECRET_COOKIE_PASSWORD,
```

Comment:
The coding convention requires us to prefix our env variables with `KS_`
```suggestion
    process.env.KS_SECRET_COOKIE_PASSWORD ?? process.env.SECRET_COOKIE_PASSWORD,
```